public/lsp/download_file.php

```diff
@@ -9,7 +9,8 @@
  * the file as a new page
  */
 function download_file($file_id, $file_name) {
-    global $DATA_DIR;
+	global $DATA_DIR;
+	$file_name = html_entity_decode($file_name);
```

Comment:
Probably better to handle this in the `GET()`, even if it's only ever used in the function.

https://github.com/LMMS/lmms.io/blob/6d0a4f113e31ef730006b0ec3bc7f4f18c7e6378/public/lsp/download_file.php#L41-L43

Reply:
So like `html_entity_decode(GET('name'))` ?

Reply:
Yeah.